vscripts/_remote_functions_mp.gnut

```diff
@@ -155,10 +155,23 @@ void function RemoteFunctions_Init_Begin()
 			Remote_RegisterUntypedFunction_deprecated( "ServerCallback_CTF_PointCaptured" )
 			Remote_RegisterUntypedFunction_deprecated( "ServerCallback_CTF_TeamText" )
 
+			//CTF: Vote Menu
+			Remote_RegisterUntypedFunction_deprecated("ServerCallback_CTF_OpenCTFVoteMenu")
+			Remote_RegisterUntypedFunction_deprecated("ServerCallback_CTF_CloseCTFVoteMenu")
+			Remote_RegisterUntypedFunction_deprecated("ServerCallback_CTF_SetVotingScreen")
+			Remote_RegisterUntypedFunction_deprecated("ServerCallback_CTF_SetNextRoundScreen")
+			Remote_RegisterClientFunction("ServerCallback_CTF_UpdateVotingMaps", "int", 0, 255, "int", 0, 255, "int", 0, 255, "int", 0, 255)
+			Remote_RegisterClientFunction("UpdateMapVotesClient", "int", 0, 255, "int", 0, 255, "int", 0, 255, "int", 0, 255)
+			Remote_RegisterClientFunction("UpdateUIVotingLocationDone", "int", 0, 255)
+			Remote_RegisterClientFunction("UpdateUIVotingLocationTied",  "int", 0, 255, "int", 0, 255)
+			Remote_RegisterClientFunction("ServerCallback_CTF_SetWinnerScreen", "int", 0, 255)
+
 			// CTF: Server to UI
 			Remote_RegisterUIFunction( "OpenCTFRespawnMenu" )
 			Remote_RegisterUIFunction( "CloseCTFRespawnMenu" )
-			
+			Remote_RegisterUIFunction( "OpenCTFVoteMenu" )
```

Comment:
Just merge OpenCTFVoteMenu and CloseCTFVoteMenu into 1 Remote call with an integer that then decides behavior via switch case.

This concept can be applied to a fuck ton of function in this gamemode..